packages/default/scss/treelist/_layout.scss

```diff
@@ -35,6 +35,17 @@
             background-color: transparent;
             background-repeat: no-repeat;
         }
+
+        tr.k-footer td,
+        tr.k-footer-template td {
+            border-width: 1px 0;
+            border-style: solid;
+            font-weight: bold;
+        }
+
+        tr.k-footer + tr.k-footer td {
```

Comment:
```tr.k-footer-template + tr.k-footer-template td ```

Should resolve the double border problem - 

![image](https://user-images.githubusercontent.com/17339722/85394938-08f14300-b558-11ea-8316-de1a68483266.png)

Reply:
Adding this rule will remove the top border from every footer row in jQuery TreeList:

<img width="701" alt="Screenshot 2020-06-30 at 13 03 23" src="https://user-images.githubusercontent.com/1826086/86113571-17001000-bad2-11ea-9159-011de7aad0ad.png">

In jQuery TreeList all aggregates are initially rendered, however their visibility is toggled via **display: none;**